x-pack/test/functional/apps/maps/add_layer_panel.js

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from 'expect.js';
+
+export default function ({ getPageObjects, getService }) {
+  const PageObjects = getPageObjects(['maps', 'common']);
+  const MapsObject = PageObjects.maps;
```

Comment:
Don't really like `MapsObject`. None of the other functional tests do this so it looks weird and inconsistent and breaks readability between test files. `MapsObject` is not much shorter than `PageObjects.maps`